src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptyforinitializerpad/InputEmptyForInitializerPad.java

```diff
@@ -0,0 +1,58 @@
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
I don't like this comments.
It was ok for shared inputs, but now they should be fixed to represent current purpose or removed